src/erlcloud_sns.erl

```diff
@@ -171,17 +173,17 @@ confirm_subscription(SnsEvent, Config) ->
     TopicArn = binary_to_list(proplists:get_value(<<"TopicArn">>, SnsEvent, <<>>)),
     confirm_subscription2(Token, TopicArn, Config).
 
--spec confirm_subscription/3 :: (sns_event(), string(), string()) -> string().
+-spec confirm_subscription/3 :: (sns_event(), parameter(), parameter()) -> string().
```

Comment:
Line 176 is too long: -spec confirm_subscription/3 :: (sns_event(), parameter(), parameter()) -> string()..